.github/workflows/stale-action.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Close stale Issues and Warn stale PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
```

Comment:
So this will run every day at 1.30AM, which to me makes more sense than every hour and a half, as you had suggested in the PR description :)

Reply:
ah really? thanks Ricky :D I clearly interpreted this wrong.

Reply:
you are right: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07 hehe